src/main/java/org/apache/cassandra/db/NoOpSchemaChangeListener.java

```diff
@@ -3,7 +3,7 @@
  *
  * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.debezium.connector.cassandra;
+package org.apache.cassandra.db;
```

Comment:
Ok, let's definitely not do that :) The split package will haunt us eventually, and it's generally not a good idea to inject code into packages not owned by us.

So taking a step back, _why_ is it that we need to unregister that MBean in the first place? If there's no public method for doing so, this makes me think it's not something we actually should do. What about invoking `ColumnFamilyStore.invalidate()`, which also handles the MBean business?

Reply:
adding a table also registers mbean but removing a table does not unregister it, so then the next adding will fail because it will try to register that mbean for the second time which fails

the only reasonable way how to do that, as what I found, is to actually unregister it - that is the proper way, but that method is package protected :)

Invocation of invalidate fails too.

Reply:
> Invocation of invalidate fails too.

Do you have any more details why it fails?